tools/downloader/README.md

```diff
@@ -29,11 +29,17 @@ future releases.
 
 ## Prerequisites
 
-1. Install Python (version 3.6 or higher)
-2. Install the tools' dependencies with the following command:
+1. Install Python (version 3.6 or higher), setuptools:
 
 ```sh
-python3 -mpip install --user -r ./requirements.in
+sudo apt-get install python3 python3-dev python3-setuptools python3-pip
+```
+
+2. Install the tools with the following command:
+
+```sh
+python3 setup.py build
+python3 setup.py install
```

Comment:
These instructions are for Ubuntu only. On Windows, if you install Python with the installer from python.org (the recommended way to install Python on Windows), setuptools and pip are included (though they are not always at the latest version). But if you type `python3` in a command prompt, you get redirected to the Windows Store to install Python. So on Windows you have to type `python`, but on Ubuntu, you have to type `python3`. 

This is how the PyPI openvino-dev install docs do this: https://github.com/openvinotoolkit/openvino/blob/master/docs/install_guides/pypi-openvino-dev.md 
For the notebooks we recommend a virtualenv and have different headings https://github.com/openvinotoolkit/openvino_notebooks/

Reply:
Changed command to `python`, added note about the difference on Linux